src/components/Input/types.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright 2024 Mia srl
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+export enum Type {
+  Outlined = 'outlined',
+  Borderless = 'borderless'
+}
+
+export enum HTMLType {
```

Comment:
I'd expect this to be named Type (no need to specify HTML) and the one above something more like Style doesn't it make more sense?

Reply:
I agree because "number" and ”password" are a subtype of input. Text, search, and hidden are not htmlType strictly

Reply:
"variant"? style doesn't seem a good idea as it creates confusion with the standard prop style imo

Reply:
>style doesn't seem a good idea as it creates confusion with the standard prop style imo

i wasn't convinced for the same reason, borderless etc are still more relevant to style than to something too generic as variant. Maybe Appearance?

Reply:
Ok, Appearance seems fine to me